nightfall-deployer/migrations/2_deploy_upgradeable.js

```diff
@@ -71,10 +72,50 @@ module.exports = async function (deployer) {
       token.amount,
     );
   }
-
-  //set Matic Address
+  // set Matic Address
   const maticAddress = RESTRICTIONS.tokens[process.env.ETH_NETWORK].find(
     token => token.name === 'MATIC',
   ).address;
   await shield.setMaticAddress(maticAddress);
+  // set initial whitelist managers
+  for (const whitelistManager of WHITELIST_MANAGERS) {
+    await shield.createWhitelistManager(whitelistManager.groupId, whitelistManager.address);
+  }
+  console.log('WHITELISTING****************', process.env.WHITELISTING);
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
+  console.log('****');
```

Comment:
:eyes: I'm not very familiar with the deployer but I wonder if we need all that or we could do with some regular logging

Reply:
Good point, I'll remove it.